types/setting.go

```diff
@@ -875,10 +875,10 @@ func ValidateSetting(name, value string) (err error) {
 
 	definition, ok := GetSettingDefinition(sName)
 	if !ok {
-		return fmt.Errorf("setting %v is not supported", sName)
+		return errors.Wrapf(err, "setting %v is not supported", sName)
```

Comment:
if `err` is nil, `errors.Wrapf()` would return nil as well. In other words, do not use this function when there is no existing error.
Similar below.

Reply:
It is my mistake to use wrap error function even there is no error returned. I will fix it asap. Thank you.